src/screens/AccountScreen.tsx

```diff
@@ -1,23 +1,22 @@
-import React, { useContext } from 'react';
+import React from 'react';
 import { View, Text } from 'react-native';
-import { Context as AuthContext } from 'src/components/context/AuthContext';
+import { useAppSelector } from 'src/redux/hooks';
+import { selectAuth } from 'src/slices/authSlice';
 import tw from 'twrnc';
 
 import Button from '../components/Button';
 
 const AccountScreen = ({ navigation }) => {
-  const { state } = useContext(AuthContext);
-
-  const { user } = state;
+  const authState = useAppSelector(selectAuth);
 
   return (
     <View style={tw`flex items-center gap-3 flex-1 bg-[#d5d7da]`}>
       <View style={tw`flex-col justify-center h-1/5`}>
-        <Text style={tw`text-3xl font-bold`}>{user.display_name}</Text>
+        <Text style={tw`text-3xl font-bold`}>{authState.display_name}</Text>
```

Comment:
again... a taste thing, but I find "user.display_name" is more clear even if it's referencing the same thing.